index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./style.css">
+  <title>CV</title>
+</head>
+<body>
+  <figure>
+   <p><img src="photo.jpg" alt="my photo" width="300" height="300"></p>
+   <figcaption><strong>Photo</strong></figcaption>
+  </figure>
+  <h1>Resume:</h1>
+  <h2>Name and surname:</h2>
+  <ul class="none-marker">
+    <li>Oleg, Konyushevsky.</li>
+  </ul>
+  <h2>Contacts:</h2>
+  <ul class="contacts none-marker">
+    <li><a href="tel:+375333017089">tel.: <em>+375333017089.</em></a></li>
+    <li><a href="mailto:olegkonyushevsky@gmail.com">email: <em>olegkonyushevsky@gmail.com.</em></a></li>
+  </ul>
+  <h2 class="clear">Summary:</h2>
+  <ul>
+    <li>Learn to write good apps.</li>
+    <li>Develop.</li>
+    <li>Feel happy about what I do.</li>
+  </ul>
+  <h2>Skills:</h2>
+  <ul>
+    <li>git</li>
+    <li>html/css</li>
+    <li>js (and node)</li>
+    <li>PHP (a little bit)</li>
+    <li>sql (a little bit)</li>
+    <li>с (a little bit)</li>
+  </ul>
+  <h2>Some piece of my program:</h2>
+  <code>

```

Comment:
It is possible to use tag `<pre>` instead of `<code>` here. 
Tag `<pre>` keep a formatting and it allows to avoid a lot of `<br>` and `<&nbsp>`.